app/ide-desktop/lib/content/src/index.ts

```diff
@@ -12,7 +12,20 @@ import GLOBAL_CONFIG from '../../../../gui/config.yaml' assert { type: 'yaml' }
 
 const logger = app.log.logger
 
+// ===================
+// === Live reload ===
+// ===================
+
+if (IS_DEV_MODE) {
+    new EventSource('/esbuild').addEventListener('change', () => {
```

Comment:
The `'/esbuild'` and `'change'` look here like magic literals. Please document.
(BTW, it's great to see this finally supported in esbuild itself.)